inlong-manager/manager-web/sql/apache_inlong_manager.sql

```diff
@@ -75,8 +75,7 @@ CREATE TABLE IF NOT EXISTS `inlong_group_ext`
     `is_deleted`      int(11)           DEFAULT '0' COMMENT 'Whether to delete, 0: not deleted, > 0: deleted',
     `modify_time`     timestamp    NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'Modify time',
     PRIMARY KEY (`id`),
-    UNIQUE KEY `unique_inlong_group_key` (`inlong_group_id`, `key_name`),
-    INDEX `group_ext_group_index` (`inlong_group_id`)
+    UNIQUE KEY `unique_inlong_group_key` (`inlong_group_id`, `key_name`)
```

Comment:
Please change the `manager-test/src/main/resources/h2/apache_inlong_manager.sql` at the same time, thanks.

Reply:
> Please change the `manager-test/src/main/resources/h2/apache_inlong_manager.sql` at the same time, thanks.

ok